src/Altinn.Platform/Altinn.Platform.PDF/src/main/java/altinn/platform/pdf/utils/TextUtils.java

```diff
@@ -41,6 +42,24 @@ public static String getTextResourceByKey(String key, TextResources textResource
     return key;
   }
 
+  /**
+   * Removes illegal chars that pdf-generation does not handle
+   * @param raw the unfiltered string
+   * @return the filtered string
+   */
+  public static String removeIllegalChars(String raw) {
+    if (raw == null) {
+      return "";
+    }
+    StringBuilder builder = new StringBuilder();
+    for (int i = 0; i < raw.length(); i++) {
+        if (WinAnsiEncoding.INSTANCE.contains(raw.charAt(i)) ) {
```

Comment:
should we replace the illegal chars with another char instead of only removing it? E.g a square or something?

Reply:
Good question. I'll have a quick chat with ux.

Reply:
Both square and � are unsupported.. After talks with ux we landed on just removing chars entirely as a first solution in order to prevent pdf-generation from breaking and fix the issue for dat.